launch_ros/launch_ros/substitutions/find_package.py

```diff
@@ -39,10 +41,16 @@ class FindPackage(Substitution):
         not found during substitution.
     """
 
-    def __init__(self, package: SomeSubstitutionsType) -> None:
+    def __init__(
+        self,
+        package: SomeSubstitutionsType,
+        *,
+        find_package_func: Callable[[str], str] = get_package_prefix
+    ) -> None:
         """Constructor."""
         super().__init__()
         self.__package = normalize_to_list_of_substitutions(package)
+        self.__find_package_func = find_package_func
 
     @classmethod
     def parse(cls, data: Iterable[SomeSubstitutionsType]):
```

Comment:
@jacobperron nit: the message in the `AttributeError` this method may raise is always the same and does not match either of the frontend tags.

Reply:
@jacobperron same for the `describe()` method, it'll look the same for both substitutions.

Reply:
See 076b334